tests/testthat/test_NAN.R

```diff
@@ -14,3 +14,44 @@ fit <- cv.iregnet(
 fit <- cv.iregnet(
   realNAN$feature.mat, realNAN$target.mat,
   family="gaussian")
+
+# Custom data which works fine
+y <- rbind(
+  c(5, 10),
+  c(6, Inf),
+  c(-Inf, 2),
+  c(1, 2),
+  c(-Inf, 3))
+
+# Custom data that produces NAN's error
+y <- rbind(
+  c(5, 10),
+  c(6, Inf),
+  c(9, Inf),
+  c(3, 4),
+  c(10, Inf))
+
+y <- rbind(
+  c(1, 2),
+  c(-Inf, 5),
+  c(3, Inf),
+  c(-Inf, 3),
+  c(-Inf, 4))
+
+test_that("Check left and right censorship", {
```

Comment:
could you add a couple of other tests for this:
1. Using a `Surv` object in which all the data points are:
  1.1. All left censored
  1.2 All right censored
2. Using `NA`s instead of `Inf` here

Reply:
The same error should still be expected

Reply:
Done :+1: